scripts/validate-commits.sh

```
@@ -0,0 +1,298 @@
+# Check if commit is a chore
+is_chore_commit() {
+  local message="$1"
+  echo "$message" | grep -qiE "chore\(|chore:"
+}
+
+# Extract commit description (look for Resolves: line anywhere in commit)
+extract_description() {
+  local message="$1"
+  
+  # First try to find a "Resolves:" line anywhere in the message (exact case)
+  local resolves_line=$(echo "$message" | grep -E "^Resolves: " | head -1)
```

Shellcheck (0.11.0)

[warning] Declare and assign separately to avoid masking return values.

(SC2155)